bug/adview_not_emitting_onadloadfailedevent (#117)
AppLovin/AppLovin-MAX-React-Native: CHANGELOG.md

## Versions

## x.x.x

>     * Fix not emitting an adLoadFailed event for a native adView (banner/mrec)

Reviewer: ```suggestion
    * Fix not emitting an `"OnBannerAdLoadFailedEvent"` or `"OnMRecAdLoadFailedEvent"` event for native UI component banners and MRECs.
```
